homeassistant/components/smartthings/sensor.py

```diff
@@ -1104,6 +1104,7 @@ class SmartThingsSensorEntityDescription(SensorEntityDescription):
     "C": UnitOfTemperature.CELSIUS,
     "F": UnitOfTemperature.FAHRENHEIT,
     "ccf": UnitOfVolume.CENTUM_CUBIC_FEET,
+    "mcf": UnitOfVolume.MILLE_CUBIC_FEET,
```

Comment:
I'm not convinced smartthings should be updated in this PR.
It feels like it should be a follow-up PR (if it's needed)

Reply:
Agreed, I'll revert that part here.